packages/web-components/src/components/va-modal/va-modal.css

```diff
@@ -0,0 +1,201 @@
+@import '../../mixins/buttons.css';
+@import '../../mixins/focusable.css';
+
+:host {
+  display: block;
+}
+
+.va-modal {
+  background: rgba(50, 58, 69, 0.8);
+  content: ' ';
+  display: block;
+  height: 100%;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 6;
+}
+
+.va-modal-inner {
+  background: var(--color-white);
+  box-sizing: border-box;
+  margin: auto;
+  max-width: 40rem;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 100%;
+  cursor: auto;
+}
+
+.va-modal-close {
+  background-color: transparent;
+  color: var(--color-primary);
+  font-size: 2.25rem;
+  padding: 0;
+  position: absolute;
+  margin: 1.6rem;
+  right: 0;
+  top: 0;
+  width: auto;
+  z-index: 9;
+  white-space: nowrap;
+}
+
+.va-modal-body {
+  margin-right: 2rem;
+  overflow-wrap: break-word;
+  padding: 2rem;
+  word-break: break-word;
+  word-wrap: break-word;
+}
+
+h1 {
+  font-family: Bitter, Georgia, Cambria, 'Times New Roman', Times, serif;
```

Comment:
sorry minor - we can use --font-serif for that https://github.com/department-of-veterans-affairs/component-library/blob/e157e3c011afb64c8431cc7c7b266ab4c786ac03/packages/web-components/src/global/variables.css#L31